src/network/clientpackethandler.cpp

```diff
@@ -1343,15 +1344,18 @@ void Client::handleCommand_HudSetSky(NetworkPacket* pkt)
 	*pkt >> skybox.bgcolor >> skybox.type >> skybox.clouds >>
 		skybox.fog_sun_tint >> skybox.fog_moon_tint >> skybox.fog_tint_type;
 
-	if (skybox.type == "skybox") {
+	if (skybox.type == "skybox" ||
+		skybox.type == "skybox_back" || skybox.type == "skybox_front") {
 		u16 texture_count;
 		std::string texture;
 		*pkt >> texture_count;
 		for (u16 i = 0; i < texture_count; i++) {
 			*pkt >> texture;
 			skybox.textures.emplace_back(texture);
 		}
-	} else if (skybox.type == "regular") {
+	}
+	if (skybox.type == "regular" ||
```

Comment:
same here. this one should probably be called `hasColors()`?

Reply:
So far the cleanest solution is to add this check somewhere directly like in `struct SkyboxParams` which seems to be working and should be possible to use code-wide.

```
bool isSkybox()
{
	return type == "skybox" || type == "skybox_back" || type == "skybox_front";
}
bool isTransparent()
{
	return type == "regular" || type == "skybox_back" || type == "skybox_front";
}
```